### ISSUE TITLE
E-mails in docblock tags are raising errors

### ISSUE THREAD

**Issue by daltones** [Gert-dev/php-integrator-linter#36]
I found the issue with this common example:
```php
/**
 * @author My Name <myname@mydomain.com>
 *                         ^^^^^^^^ Error: mydomain was not found.
 */
```

But it seems like `php-integrator-linter` (or `-base`, I don't known exactly how it works) is not handling correctly all `@`s inside docblock tags. I guess it thinks that every `@` will start a new tag.

**Comment by Gert-dev** [comment]
Seems to be a bug in the base package (the base package does the actual linting and this package uses a method from the base package, in case you were wondering).

**Comment by Gert-dev** [comment]
Thanks for reporting this, it should be fixed in base 1.0.6. Could you confirm?

**Comment by daltones** [comment]
Fixed! 😄 Thanks =)
